packages/backend/src/services/User.js

```diff
@@ -72,36 +72,68 @@ const query = {
     telephone,
     status_code as "statusCode"
   FROM front.users
-  WHERE 
+  WHERE
     mail = $1
-    AND pwd = crypt($2, CASE 
-      WHEN pwd = '' 
-      THEN 
-        gen_salt('bf') 
-      ELSE 
-        pwd 
-      END 
+    AND pwd = crypt($2, CASE
+      WHEN pwd = ''
+      THEN
+        gen_salt('bf')
+      ELSE
+        pwd
+      END
     )
     AND deleted is False
   `,
   select: (criterias) => [
     `
-      SELECT 
-        id,
-        mail as email,
-        pwd is not null as "hasPwd",
-        nom,
-        prenom,
-        telephone,
-        status_code as "statusCode"
-      FROM front.users
-      WHERE 1=1 
+      SELECT
+        us.id as "id",
+        us.mail as email,
+        us.pwd is not null as "hasPwd",
+        us.nom as "nom",
+        us.prenom as "prenom",
+        us.telephone as "telephone",
+        us.created_at as "createdAt",
+        us.status_code as "statusCode",
+        o.personne_morale->>'siret' as "siret"
+        o.personne_morale->>'raisonSociale' as "raisonSociale"
+      FROM front.users us
+      INNER JOIN front.user_organisme uo ON us.id = uo.use_id
```

Comment:
A priori, un user est lié a un seul organisme, donc ok. Cela dit, cette table de relation 'a aucun sens, elle ne devrait etre utilisée que dans des relations many to many ce qui n'est pas le cas ici.

Ce serait pas mal de créer un ticket de dete technique : le relation du user a l'orgaisme devrait etre dans la table user

Reply:
Oui je suis d'accord j'ai pas trop compris sur le moment mais je me suis dit qu'il devait y avoir une raison que je ne connaissais pas, je vais créér un ticket dans dette technique

Reply:
on a vérifié sur dev et prod, on a aucun user lié a plusieurs organismes